client/src/app/shared/models/motions/motion.ts

```diff
@@ -54,7 +54,9 @@ export class Motion extends AgendaBaseModel {
     }
 
     public getTitle(): string {
-        return this.title;
+        if (this.identifier) {
```

Comment:
would return undefined if there is no identifier. Should return `this.title` if there is no identifier, no?